app/Http/Services/PurchaseOrderService.php

```
@@ -0,0 +1,39 @@
+            return RequestPurchaseOrder::create([
+                'transaction_date' => now(),
+                'po_number' => static::generatePoNumber(),
+                'request_canvass_summary_id' => $requestCanvassSummary->id,
+                'name_on_receipt' => null,
+                'delivered_to' => null,
+                'processing_status' => PurchaseOrderProcessingStatus::PENDING,
+                'metadata' => $requestCanvassSummary->metadata ?? [],
+                'created_by' => $requestCanvassSummary->created_by,
+                'request_status' => $requestCanvassSummary->request_status,
+                'approvals' => $requestCanvassSummary->approvals,
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\RequestPurchaseOrder' in method 'createPurchaseOrderFromCanvass'. (Clean Code Rules)

(StaticAccess)